### ISSUE TITLE
Awkward padding in sync popup

### ISSUE THREAD

**Issue by user1823** [ankidroid/Anki-Android#15351]
In alpha 17, the sync popup has a somewhat awkward padding.

<img width=300 src="https://github.com/ankidroid/Anki-Android/assets/92206575/cfed5913-b34b-4f4d-9705-43d560bf9271">

**Comment by criticalAY** [comment]
https://github.com/ankidroid/Anki-Android/issues/15346#issuecomment-1913134361

**Comment by BrayanDSO** [comment]
I'd also add that the progress dialog doesn't have square corners, so I upgraded the issue to matching the style of progress dialogs with the style of the other dialogs

**Comment by criticalAY** [comment]
> progress dialog doesn't have rounded corners

Fixed it for all dialogs

**Comment by BrayanDSO** [comment]
> Progress dialog  - 
![image](https://github.com/ankidroid/Anki-Android/assets/48384865/ab7c774a-d095-431e-b130-b2f1a01b6fcd)


The screenshot in your PR shows the same square corners

**Comment by criticalAY** [comment]
I haven't updated it because the buttons color is still pending

**Comment by criticalAY** [comment]
https://github.com/ankidroid/Anki-Android/pull/15350#issuecomment-1913511741

**Comment by BrayanDSO** [comment]
@user1823, is this still the case?

**Comment by user1823** [comment]
Yes, it is still asymmetrical. But, the corners are now rounded. 

<img width=350 src="https://github.com/user-attachments/assets/75638132-ca42-4352-9ee5-4b93a6c909f7">

**Comment by Ayush-Patel-56** [comment]
I think its the by default interface as mentioned by @criticalAY in comment. We are good to close this issue if it's fixed.
